batch/az-create-worker-image.sh

```diff
@@ -17,7 +17,7 @@ BUILD_IMAGE_RESOURCE_GROUP="${RESOURCE_GROUP}-build-batch-worker-image"
 VM_NAME=build-batch-worker-image
 WORKER_VERSION=0.0.12
 
-USERNAME=$(az ad signed-in-user show --output tsv --query mailNickname)
+USERNAME=$(whoami)
```

Comment:
This seems dangerous, it might not be the case that, e.g. the Australians have the same usernames locally as in AZ.

Reply:
I think I might be misunderstanding exactly how we authenticate. AFAIK now, azure credentials aren't connected to SSH'ing into azure VMs the way that it is in GCP. When creating a VM with the `--generate-ssh-keys` flag (which honestly I'm thinking we maybe shouldn't use), Azure uses your `~/.ssh/id_rsa.pub` public key as a trusted key for the vm, and creates one if it doesn't exist. So the following ssh command should use `~/.ssh/id_rsa` and should work for the current user.

Reply:
oh, dang, OK. In that case `whoami` seems exactly right.